deploy/helm/sumologic/templates/service-headless.yaml

```diff
@@ -3,11 +3,11 @@ kind: Service
 metadata:
   name: {{ template "sumologic.fullname" . }}-fluentd-headless
   labels:
-    app: {{ template "sumologic.labels.app" . }}
+    app: {{ printf "%s-fluentd-headless" (include "sumologic.labels.app" .) }}
```

Comment:
I think we should stick to the templates :)
We could create templates for events, fluentd, otelcol and whatever else we have. for both names and labels

Reply:
That's a good point but let's do it in 1.1 and not change the 1.0 scope any further.
I've created an issue to keep track of that one: https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/574